performance/setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+if type mysql >/dev/null 2>&1; then
+    echo "MySQL present."
+    export MYSQLHOST=localhost
+    export MYSQLUSER=root
+    export MYSQLPORT=3306
+    #update MySQL password before running the benchmark
+    export MYSQLPWD=
+    gcloud auth application-default login
```

Comment:
Nit: Add a print statement after this that login was successful and records are going to be inserted

Reply:
added